Fst2List.cpp

```diff
@@ -2416,7 +2875,7 @@ int main_Fst2List(int argc, char* const argv[]) {
         aa.traitAuto = MULTI;
         break;
       default:
-        error("Invalid arguments: rerun with --help\n");
+        error("Invalid arguments : rerun with --help\n");
```

Comment:
No space is required before the colon.

Reply:
For comments or code refactoring, it would be preferable to open a different PR.

Reply:
OK, I will do that next time I have comments to change or code to refactor